### PR TITLE
Resolves #121: add vpg performance counters

### LINKED ISSUE #121: Add VpgResource.getPerformanceCounters()

### PR DESCRIPTION
**Pull request by nksprzak**

### CODE REVIEW

**Comment by cfsnyder** [comment]
you have conflict to resolve @nksprzak

File: src/sdk/model/common/__json__/performance-counter-json.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**
+ * Interface for API performance counter JSON representation.
+ */
+export interface PerformanceCounterJson {
```
Comment (cfsnyder) [inline src/sdk/model/common/__json__/performance-counter-json.ts]: add to index

Comment (nksprzak) [reply src/sdk/model/common/__json__/performance-counter-json.ts]: fixed

File: src/sdk/model/vpg/vpg.ts [hunk 2]
```diff
@@ -471,4 +473,16 @@ export class Vpg extends Entity {
     const observable: Observable<string> = Iland.getAuthProvider().getTokenObservable();
     return observable.map(token => `${href}&oauth_token=${token}`);
   }
+
+    /**
+     * Get performance counters for vpg
```
Comment (cfsnyder) [inline src/sdk/model/vpg/vpg.ts]: last thing.. we try to standardize on ending the description in jsdoc with a period.

**Comment by cfsnyder** [comment]
Actually, I just realized that we already have PerfCounter model objects in the mixins folder, so we'll need to re-use those instead of creating new ones

**Comment by cfsnyder** [comment]
thanks Nick

**Pull request merged**